### ISSUE TITLE
Nextvi new development

### ISSUE THREAD

**Issue by kyx0r** [aligrudi/neatvi#89]
Hello Ali,

A few months back I've resumed active development of Nextvi and with that I've managed to solve
quite a lot of old problems and improve existing features further. For example, bounded rendering
of syntax highlighting in reverse, with arbitrarily reversed text. The highlighting will be consistent
and bounded in any direction.

I've also written a comprehensive manual for Nextvi which covers basically every aspect.
Some users might find it useful to learn basic vi features there or to try Nextvi.
I'm leaving this the link here. Publicly visible on github as well.
https://github.com/kyx0r/nextvi/tree/master

It's nothing special, but I hope it helps to revisit things.

Kind Regards,
Kyryl

**Comment by kyx0r** [comment]
@aligrudi 
Hello Ali, I still work on Nextvi and close to 5.0 spec getting released. If take a look if you're interested because it has come a long way.

In short, I have made a robust patching system in Nextvi that uses Nextvi, and pushes the bar further to support a lot more optional patches that the users can use. This way, users are not locked into a particular implementation except the core Nextvi specification. This keeps it small, prevents unwanted feature creep, and makes it easy to understand the essentials.

To clarify, the kind of features I'm working on are not trivial: visual mode, lsp support, window management, etc. And all this supported by patching the software. Not forced down your throat at the will of the developer. In plain terms, Suckless philosophy with a twist, addressing where it normally falls flat - compatibility between versions and patches.

Long story short, I am working on something very important that hasn't been done before. And the way it is done, pretty much unheard of. Solving the compatibility (e.g divergence problem) at patch time.

To anyone else reading - join me. I need to see what people can do with the software this far. Nextvi will be the only editor that is truly configurable to the lowest level. Send your customization for review to be added to patches branch.

**Comment by aligrudi** [comment]
Hi,

Kyryl ***@***.***> wrote:
> Good luck maintaining that. It will be a while before it can reach
> the quality level of Nextvi, but it is a start. The power of Nextvi
> comes from useful examples and test cases that actually work and never
> have any bugs. Some Neatvi features are not well designed, they are
> too rigid and will require changing in the future and some are
> completely undocumented.

As this is not a real Neatvi issue, I am closing it.  And, about
quality or design, I certainly disagree with you.

Dear Kyryl, such messages are more suitable in the issue tracker of
your fork, not Neatvi's.  If you experience any specific issue as a
Neatvi user, please submit it here and include enough details so that
it could be reproduced.

	Ali
--
closes #89.

**Comment by kyx0r** [comment]
@aligrudi Hi Ali, sorry if my messages seemed inappropriate or rude in any way. You're right, this is probably not the place to post. As such and given your disinterest, I am closing this issue on github as well. Not sure if you're aware, but it is no longer possible to close github issues and PRs on github through email alone.

Just in case: you can still email me privately.

Best wishes, and thanks for making Neatvi. Without it, I wouldn't be where I'm at. It does get some things right after-all.
